src/js/angular/core/templates/inactive-plugin-warning-page.html

```diff
@@ -3,9 +3,8 @@
         {{'inactive.plugin.warning.for.use' | translate}}
     </p>
     <button class="btn btn-primary btn-lg confirm-btn"
-            uib-tooltip="Activate {{humanReadablePluginName}}"
+            gdb-tooltip="Activate {{humanReadablePluginName}}"
             tooltip-placement="top"
```

Comment:
As I can see, the default position is `top`, can we omit this config where the default is needed?

Reply:
Yes we can. I was wondering if I want to remove them. I didn't change them because, the default before was also top and those are explicitly set.